### PR TITLE
Replace Object formatting with text

### PR DESCRIPTION
**Pull request by wldcordeiro**
Issue #594 - Not 100% sure I exposed the prop right so some review is nice.

### CODE REVIEW

File: public/js/components/ObjectInspector.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ const ObjectInspector = React.createClass({
     let objectValue;
     if (nodeHasProperties(item) || nodeIsPrimitive(item)) {
       const object = item.contents.value;
-      objectValue = Rep({ object });
+      objectValue = Rep({ object, mode: "tiny" });
```
Comment (wldcordeiro) [inline public/js/components/ObjectInspector.js]: Not sure I always want tiny, maybe I do? Clarify @jasonLaster

File: public/js/components/Rep.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ const Grip = require("../lib/devtools/client/shared/components/reps/grip").Grip;
 
 require("../lib/devtools/client/shared/components/reps/reps.css");
 
-function renderRep({ object }) {
-  return Rep({ object, defaultRep: Grip });
+function renderRep({ object, mode }) {
```
Comment (wldcordeiro) [inline public/js/components/Rep.js]: Should I have a default mode value?

File: public/js/components/ObjectInspector.js (hunk 1 above)
Comment (jasonLaster) [inline public/js/components/ObjectInspector.js]: I'm not sure either, I think we probably do... 

Lets take a look at the tests and see if everything makes sense
https://dxr.mozilla.org/mozilla-central/source/devtools/client/shared/components/test/mochitest/test_reps_object.html#83

File: public/js/components/Rep.js (hunk 2 above)
Comment (jasonLaster) [inline public/js/components/Rep.js]: Hmm, whats the default behavior if undefined is passed in?

File: public/js/components/ObjectInspector.js (hunk 1 above)
Comment (jasonLaster) [inline public/js/components/ObjectInspector.js]: based on the tests tiny looks fine. @linclark thoughts?

File: public/js/components/Rep.js (hunk 2 above)
Comment (wldcordeiro) [inline public/js/components/Rep.js]: Looks like it uses the default output? https://dxr.mozilla.org/mozilla-central/source/devtools/client/shared/components/test/mochitest/test_reps_object.html#79

Comment (jasonLaster) [inline public/js/components/Rep.js]: 👍

**Comment by jasonLaster** [comment]
👍 

Once this lands we can create an m-c issue for updating the Object Inspector

File: public/js/components/ObjectInspector.js (hunk 1 above)
Comment (linclark) [inline public/js/components/ObjectInspector.js]: @jasonLaster I don't know enough about the desired output across the tree and the way reps are being used here to have an opinion.

Comment (wldcordeiro) [inline public/js/components/ObjectInspector.js]: I'll leave this to you @jasonLaster heh, I'm in the same boat as @linclark though it does seem to look like your expect screenshot.

Comment (jasonLaster) [inline public/js/components/ObjectInspector.js]: Thanks @wldcordeiro. I'm okay moving forward with this and adjusting if we need to.

**Comment by jasonLaster** [comment]
Thanks @wldcordeiro!

**Pull request merged**